src/sentry/reprocessing2.py

```diff
@@ -154,11 +156,15 @@ def backup_unprocessed_event(project, data):
     event_processing_store.store(dict(data), unprocessed=True)
 
 
-def reprocess_event(project_id, event_id, start_time):
+@dataclass
+class ReprocessableEvent:
+    event: Event
+    data: Dict[str, Any]
+    attachments: List[models.EventAttachment]
 
-    from sentry.ingest.ingest_consumer import CACHE_TIMEOUT
+
+def pull_event_data(project_id, event_id) -> ReprocessableEvent:
     from sentry.lang.native.processing import get_required_attachment_types
-    from sentry.tasks.store import preprocess_event_from_reprocessing
 
     with sentry_sdk.start_span(op="reprocess_events.nodestore.get"):
```

Comment:
Now that `event` is checked before `data`, we can move this span down to after the event check, right?